tests/scenario_tests/test_app.py

```diff
@@ -77,7 +74,7 @@ class TestExecutor(Executor):
 
     def test_valid_single_auth(self):
         app = App(signing_secret="valid", client=self.web_client)
-        assert app != None
+        assert app is not None
```

Comment:
good catch

Reply:
True, but I think I'll remove these changes in order to simplify the PR